Mathlib/CategoryTheory/Comma/Basic.lean

```diff
@@ -257,7 +257,8 @@ instance faithful_map [F₁.Faithful] [F₂.Faithful] : (map α β).Faithful whe
 
 instance full_map [F.Faithful] [F₁.Full] [F₂.Full] [IsIso α] [IsIso β] : (map α β).Full where
   map_surjective {X Y} φ :=
-   ⟨{ left := F₁.preimage φ.left
+    ⟨{
+      left := F₁.preimage φ.left
```

Comment:
There's somthing similar in `essSurj_map` (right after this), right? Maybe we should change all of those at once.